### ISSUE TITLE
Pics don't load from unblocked domain

### ISSUE THREAD

**Issue by tegru** [RequestPolicy/requestpolicy#455]
Affected site. iltasanomat.fi
Allowed destination: snstatic.fi

After installing RequestPolicy some pictures don't load (especially from sub domain is12.snstatic.fi) even though the required cross-site request is allowed.

The strange part is that even uninstalling RequestPolicy doesn't fix this at all...

Before using RP I've had this exact same problem with Adblock Edge. Only new browser profile fixed it... Adblock Plus doesn't have this problem.

**Comment by myrdd** [comment]
Hi @tegru I couldn't reproduce your problem. You might want to try RPC: https://requestpolicycontinued.github.io/

> The strange part is that even uninstalling RequestPolicy doesn't fix this at all...

If this is the case, it's probably not an issue with RP.

**Comment by tegru** [comment]
@myrdd Yeah, turns out it was actually HTTPS-Everywhere enabling a non-working https-connection for snstatic.fi...
